libqtile/backend/x11/core.py

```diff
@@ -638,6 +655,20 @@ def handle_PropertyNotify(self, event) -> None:  # noqa: N802
             self._selection[name]["selection"] = value
             hook.fire("selection_change", name, self._selection[name])
 
+        elif all(
+            (
+                name in ("ESETROOT_PMAP_ID", "_XROOTPMAP_ID"),
+                event.window == self._root.wid,
+                self.qtile.config.x11_fake_transparency,
+            )
+        ):
+            self.root_pixmap = self._get_root_pixmap()
```

Comment:
If only we'd typed the X backend... the return type of `self._get_root_pixmap()` is `None` ;)

Reply:
Interesting. I wonder why it still worked!

Reply:
Honestly me too, surely it's always None?

Reply:
Ah. It's because the drawers check this (correctly) if the value is None.